vagrant/boxes.d/30-source.yaml

```diff
@@ -4,7 +4,8 @@ pulp3-source-fedora32:
     host_path: '..'
     guest_path: '/home/vagrant/devel'
     reverse: False
-  memory: 4096
+  memory: 6144
+  cpus: 4
```

Comment:
I agree on this compromise.

It may be a blip, as the test has taken even longer once, but the CentOS 8 Source FIPS CI test took longer than usual.

Please do the following:
1. Put the old `memory` and `cpus` in .github/files/qemu/settings.yaml . The CI uses this.
2. Remove my should-never-have-done-them-this-way `sed` settings in .github/workflows/scripts/script.sh .  The only 2 lines beginning with `sed`

Reply:
done